packages/notifier/test/test-stored-subscription.js

```diff
@@ -80,3 +86,56 @@ test('stored subscription', async t => {
   await E(publication).finish(terminalNull);
   await check('terminal null', terminalNull, true);
 });
+
+test('stored subscriber', async t => {
+  t.plan((jsonPairs.length + 2) * 4 + 1);
+
+  const initialValue = 'first value';
+  const { publication: pubStorage, subscription: subStorage } =
+    makeSubscriptionKit();
+  const storage = makeFakeStorage('publish.foo.bar', pubStorage);
+
+  const { publisher, subscriber } = makePublishKit();
+
+  publisher.publish(initialValue);
+  const storesub = makeStoredSubscriber(subscriber, storage);
+
+  t.deepEqual(await E(storesub).getStoreKey(), await E(storage).getStoreKey());
+  const unserializer = E(storesub).getUnserializer();
+
+  const notifier = makeNotifierFromSubscriber(subscriber);
+  const ait = E(notifier)[Symbol.asyncIterator]();
+  const storeAit = subStorage[Symbol.asyncIterator]();
```

Comment:
I find these declarations a bit confusing. Suggestion:
```suggestion
  const storedSubscriberIterable = subscribeEach(subscriber);
  const storedSubscriberIterator = E(storedSubscriberIterable)[Symbol.asyncIterator]();
  const storageIterator = subStorage[Symbol.asyncIterator]();
```